Pod/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -530,25 +528,35 @@ - (void)applicationWillTerminate
 {
     [self dispatchBackgroundAndWait:^{
         if (self.queue.count)
-
-            [self persistQueue];
+            [[self.queue copy] writeToURL:self.queueURL atomically:YES];
     }];
 }
 
+-(void)applicationDidFinishLaunching
```

Comment:
I added this application state change event so we would check if there is a previous version of our SDK and then we would check if there's an existing queue/traits in NSUserDefaults and if it exists, then copy it to disk and delete the copy from NSUserDefaults. I'm not sure this will work and thinking there's a better way to do this so let me know what you think @tonyxiao @f2prateek